docs/cloud-tutorials/cicd.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 7
+title: CI/CD tools
+---
+
+This this section we will cover tools provided by DBOS Cloud to enable CI/CD best practices.
+
+## Environments
+Use a different DBOS Cloud application for each environment, _e.g._, dev, staging and production.
+You can do so by [deploying](application-management#deploying-applications) the same application multiple times, each time with a different name, _e.g._, `my-app-dev`, `my-app-staging`, `my-app-prod`.
+
+:::info
+You must use a different database name (`app_db_name`) for each application.
+:::
+
+## Secrets management
+The recommended way to make secrets, such as API keys, available to your application running in DBOS Cloud is to use environment variables.
+You can use the application's `dbos-config.yaml` [environment section](../api-reference/configuration#environment-variables) to do so.
+
+`dbos-config.yaml` supports setting variables values from the environment. For instance:
+
+```yaml
+environment:
+    API_KEY: ${API_KEY}
+```
+
+Will be resolved to:
+```yaml
+environment:
+    API_KEY: "your-secret-value"
+```
+
+If `API_KEY` is set to `your-secret-value` in the CI/CD client's environment.
```

Comment:
Move this sentence before the code block?

```suggestion
If `API_KEY` is set to `your-secret-value` in the CI/CD client's environment, your config file will be updated in DBOS Cloud: 
```yaml
environment:
    API_KEY: "your-secret-value"
```
```